### DISCUSSION TITLE
Activate the "Font size" option in the Table plugin

### DISCUSSION THREAD

**Discussion by Daniel-Mompean** [grafana/grafana#38136] [Dashboards, Panels & Visualizations]
**What would you like to be added**: Activate the "Font size" option in the Table plugin, ex: 60%, 80%, 100%, 120%, etc, in previous versions it was possible.

**Why is this needed**: in some cases we want to work on the sizing of the indicators on the screen (without scroll), this option helps a lot

**Comment by therealcodlord** [comment]
+1 for this missing feature to be added to new table panel, but @Daniel-Mompean if you didn't know you can continue to use the old v6 table panel in v7, see here for details: https://community.grafana.com/t/table-transform-json-data-feature-missing-in-new-table-plugin/31559

**Comment by Daniel-Mompean** [comment]
1- i changed Visualization to Table-old, but Font size don't work

2- i would like to use Font size (v6) and hide header (v7) in the same table

**Comment by nickcuper** [comment]
any updates, it would be nice to control font size

**Comment by scurvy-dog** [comment]
I am also curious on the status if this feature request is being worked on. Being able to size table data allows for customization based on multiple monitor sizes for readability.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Reply by douginoz** [reply]
Any update on this?  I have tables that use up huge amounts of space unnecessarily and would like to shrink the font within them.  I don't understand why grafana insists on forcing large fonts for cells yet will use smaller fonts for titles, etc.  I end up with status screens that are pages in length that could easily fit in one screen if I could control how big the fonts are.  I know I can tell my browser to shrink down the entire page but this isn't a solution and distorts otherwise perfectly fine graphs and tables.

**Comment by gelicia** [comment]
Hello! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

This discussion was converted into an issue, which is available [here](https://github.com/grafana/grafana/issues/73438).
